cluster/ci/monitoring/Makefile

```diff
@@ -1,4 +1,5 @@
 ### --as=system:serviceaccount:ci:config-updater
+prometheus-k8s_token := $(shell oc get secret -n openshift-monitoring grafana-datasources -o json | jq -r '.data["prometheus.yaml"]' | base64 -d | jq '.datasources[].name = "prometheus-k8s"' | jq '.datasources[].basicAuthPassword')
```

Comment:
Prefer

```
oc get secret -n openshift-monitoring grafana-datasources -o "jsonpath={.data.prometheus\.yaml}" --as system:admin | base64 --decode | jq '.datasources[] | select(.name = "prometheus-k8s") .basicAuthPassword'
```

Reply:
I didn't use the `--as system:admin` because the postsubmit is using the `config_updater` rbac. That's why I add an extra role/rolebinding to the `openshift-monitoring` namespace, to allow a `get` `secrets`.

Reply:
I didn't mean to use `--as system:admin` but the more intelligent JQ and JSONPATH expression instead of shelling out a lot

Reply:
addressed.